content/blog/22-23/12urenloop.md

```diff
@@ -0,0 +1,120 @@
+
+De Raspberry Pi's in de stations luisteren naar de beacon-pakketjes die uitgestuurd worden door de batons en slaan de informatie uit de ontvangen pakketten op in een lokale database:
+
+- MAC-adres van de baton
+- RSSI (signaalsterkte, dit komt kort door de bocht overeen met de afstand van de baton tot het station)
+- uptime en batterijpercentage van de baton
+- Unix timestamp waarop het pakket ontvangen is
+- het ID van de detectie (een getal dat strikt stijgt, per station)
+
+Vroeger stuurden de Gyrids die gegevens dan rechtstreeks door naar count-von-count. Hierdoor was de setup
+zeer gevoelig aan netwerk- en ander falen: als er bijvoorbeeld een koets over de netwerkkabels reed (helaas is dit geen theoretisch voorbeeld) of count-von-count down was, dan gingen alle detecties verloren tot de detecties terug toekwamen. Dit is nu opgelost door de detecties vanaf Telraam binnen te trekken in plaats van die vanuit de stations naar Telraam te duwen: er draait een kleine webserver op elk station die een endpoint `/detections/<id>` heeft. Deze geeft alle detecties *na* het meegegeven ID terug: op die manier is het makkelijk
+om alle detecties naar een lokale databank te syncroniseren. Als extra voordeel is het hiermee nu ook mogelijk
+om voor elk station alle detecties te downloaden waarmee dan het event gereplayed kan worden. Zo kan er makkelijk gesleuteld worden aan het algoritme in Telraam, zonder telkens een fysiek event te moeten doen.
+
+De spanningsomvormers (buck converters) zetten het voltage van de batterij om in 9V voor de switch en 5V
+voor de Raspberry Pi. Ze zijn via een JST-connector verbonden met de autobatterij, waardoor het niet meer 
+mogelijk is om per ongeluk de polariteit om te keren. Aan een van de omvormers hangt ook een digitale 
+spanningsmeter, waardoor het in één oogopslag duidelijk is hoeveel energie er nog in de autobatterij zit.
+Ter bescherming van de omvormers zijn er ook behuizingen ge-3D-print.
+
+<p float="left">
+  <img src="https://pics.zeus.gent/aZT3dvn424rHhjmtCBkAJUFX28uOZmZCsGMFjOJW.jpg" width="49%" />
+  <img src="https://pics.zeus.gent/rbCTBUH1t03klQs9oLlU0dMNudlrzGEHrmnmd6b1.jpg" width="49%" /> 
+</p>
+
+Op onderstaande foto kan je zien wat er allemaal in een station-doos gaat (behalve de netwerkswitch of ethernetkabels)
+
+![Inhoud van de stations](https://pics.zeus.gent/MFnFeInrzspGuW2CTXzxTmx1ggxVj3xemVZoTjmk.jpg)
+
+# Netwerk
+
+Ook qua netwerk zijn we er wat op vooruit gegaan: vroeger trokken we voor onze uplink een kabel 
+van de nabijgelegen Sint-Pietersabdij naar de IT-container. De vorige editie hebben we deze 
+kabel vervangen door een "straler" (een point-to-point Wi-Fi verbinding die zich gedraagt als 
```

Comment:
```suggestion
kabel vervangen door een "straler" (een point-to-point wifi-verbinding die zich gedraagt als 
```

Reply:
Hm, ik weet niet of ik hier mee akkoord ben, na een thesis over Wi-Fi te doen, wil ik dat juist schrijven enzo

Reply:
Dat is natuurlijk deels een kwestie van persoonlijke smaak, maar het is niet dat ik dat heb uitgevonden: https://www.vlaanderen.be/taaladvies/taaladviezen/wifi-wifi-wifi-wi-fi :)

(maar het blijft een grijze zone, ik denk dat Wi-Fi niet verkeerd is)